app/components/open_project/common/submenu_component.sass

```diff
@@ -14,6 +14,10 @@
     flex-direction: column
     row-gap: 16px
 
+    input::placeholder,
+    .FormControl-input-leadingVisual
+      color: color-mix(in srgb, var(--main-menu-font-color) 80%, white)
```

Comment:
What is the reason for that rule instead of using `var(--main-menu-font-color)` directly?